linera-sdk/src/contract/mod.rs

```diff
@@ -41,18 +41,18 @@ macro_rules! contract {
                 &mut self,
```

Comment:
@jvff Shouldn't this have been a method with a default implementation in the `Contract` trait itself (or an extension)?

Reply:
Good point. I can try that :+1: